lib/src/http_request/rust_http_queue_requester.rs

```diff
@@ -80,10 +85,21 @@ impl RustHttpQueueRequester {
         url: GString,
         method: godot::engine::http_client::Method,
         body: PackedByteArray,
-        headers: VariantArray,
+        headers: Dictionary,
     ) -> Gd<Promise> {
         self._request_json(url, method, Some(body.to_vec()), headers)
     }
+
+    // #[func]
+    // fn try_set_global_network_inspector(&mut self) {
+    //     let Some(sender) = DclGlobal::try_singleton()
+    //         .and_then(|v| Some(v.bind().get_network_inspector().bind().get_sender()))
+    //     else {
+    //         return;
+    //     };
+
+    //     self.http_queue_requester = set_inspector_sender(sender);
+    // }
```

Comment:
```suggestion
```

should we remove this commented code? otherwise I would add a comment to know what it does and why is commented